pytket/CMakeLists.txt

```diff
@@ -56,7 +56,8 @@ target_link_libraries(circuit PRIVATE
     tket-Routing
     tket-Simulation
     tket-Utils)
-target_link_libraries(circuit PRIVATE ${CONAN_LIBS_SYMENGINE})
+target_link_libraries(circuit PRIVATE
```

Comment:
Have you thought about defining a variable for the libraries that this only has be be changed in one line for the hole file in future changes?

Reply:
I will do that.

Reply:
Done.